methods/secular/Secular_Requirement_Validation.ipynb

```diff
@@ -9,7 +9,8 @@
     "**Original code authored by:** David Bekaert, Heresh Fattahi, Eric Fielding, and Zhang Yunjun \n",
```

Comment:
<p>What does this do if there is no ionospheric correction available?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>